lib/tests/run_tests.js

```diff
@@ -140,9 +140,14 @@ module.exports = {
               console.log(`Coverage report created. You can find it here: ${fs.dappPath('coverage/__root__/index.html')}\n`);
               const opn = require('opn');
               const _next = () => { next(); };
-              opn(fs.dappPath('coverage/__root__/index.html'), {wait: false})
-                .then(() => new Promise(resolve => setTimeout(resolve, 1000)))
-                .then(_next, _next);
+              if (!options.noBrowser) {
+                opn(fs.dappPath('coverage/__root__/index.html'), {wait: false})
+                  .then(() => new Promise(resolve => setTimeout(resolve, 1000)))
+                  .then(_next, _next);
+              } else {
+                new Promise(resolve => setTimeout(resolve, 1000))
```

Comment:
No need for a promise here or a timeout. The timeout was just to leave time for opn to open the browser. Just need to callback here (using `_next`).